src/Response.php

```diff
@@ -23,30 +23,30 @@ class Response
     protected $status = null;
 
     /**
-     * Error code
+     * Error codes
      *
-     * The error code if the status is false. The different error codes can be found in the
+     * The error codes if the status is false. The different error codes can be found in the
      * recaptcha API docs.
      *
-     * @var string
+     * @var array
      */
-    protected $errorCode = null;
+    protected $errorCodes = [];
 
     /**
      * Class constructor used to construct a response
      *
      * @param string $status
-     * @param string $errorCode
-     * @param HTTPResponse $httpResponse If this is set the content will override $status and $errorCode
+     * @param array $errorCodes
+     * @param \Zend\Http\Response $httpResponse If this is set the content will override $status and $errorCode
```

Comment:
Use the same typehint as found in the signature; IDEs and phpdoc will expand them correctly.